security/v1/types.go

```diff
@@ -93,6 +93,13 @@ type SecurityContextConstraints struct {
 	AllowHostPID bool `json:"allowHostPID" protobuf:"varint,11,opt,name=allowHostPID"`
 	// AllowHostIPC determines if the policy allows host ipc in the containers.
 	AllowHostIPC bool `json:"allowHostIPC" protobuf:"varint,12,opt,name=allowHostIPC"`
+	// AllowHostUsers determines if the policy allows host users in the containers.
+	// If unspecified, it will be interpreted as "Allowed".
+	// +openshift:enable:FeatureGate=UserNamespacesSupport
+	// +kubebuilder:validation:Enum="Allowed";"Disallowed";""
+	// +kubebuilder:validation:Optional
+	// +optional
+	AllowHostUsers AllowHostUsersStrategyType `json:"allowHostUsers" protobuf:"varint,26,opt,name=allowHostUsers"`
```

Comment:
I would make this into
```suggestion
	// hostUsers determines if the policy allows host users in containers.
	// Valid values are "Allowed", "Disallowed" and omitted.
	// When omitted, this means no opinion and the platform is left to choose a reasonable default.
	// The current default is "Allowed".
	// +openshift:enable:FeatureGate=UserNamespacesSupport
	// +kubebuilder:validation:Enum="Allowed";"Disallowed";""
	// +kubebuilder:validation:Optional
	HostUsers HostUsersStrategyType `json:"hostUsers" protobuf:"varint,26,opt,name=allowHostUsers"`
```

Reply:
done!

Reply:
@rphillips WDYT about this? your comment contradicts...

Reply:
https://github.com/openshift/enhancements/pull/1649#discussion_r1678338441